nvflare/private/fed/client/client_json_config.py

```diff
@@ -30,7 +30,7 @@ def __init__(self):
 
 
 FL_PACKAGES = ["nvflare"]
-FL_MODULES = ["server", "client", "aggregators", "handlers", "pt", "app", "app_common", "workflows"]
+FL_MODULES = ["apis", "app_common", "widgets"]
```

Comment:
This should be fine, but let's verify and make sure that existing examples are not broken.